horreum-web/src/domain/schemas/SchemaList.tsx

```diff
@@ -33,22 +33,22 @@ export default function SchemaList() {
     const isTester = useTester()
 
     const removeSchema = (id: number) => {
-        if ( schemaCount > 0 ){
+        if (schemaCount > 0) {
             setSchemas(schemas?.filter(schema => schema.id !== id) || [])
-            const newCount = (schemaCount == 0) ? 0 : schemaCount - 1
+            const newCount = schemaCount == 0 ? 0 : schemaCount - 1
             setSchemaCount(newCount)
         }
     }
 
     const reloadSchemas = () => {
         setLoading(true)
         schemaApi
-            .list('Ascending', pagination.perPage, pagination.page - 1)
-            .then((result) => {
+            .list(pagination.perPage, pagination.page - 1, "name")
```

Comment:
Looking at the api the third argument is sort by column afaik, not sort direction. Changed it to name, but would like another opinion on it as well.

Reply:
lgtm, in the situation the third argument is empty then `name` is set as the assigned [value](https://github.com/Hyperfoil/Horreum/blob/master/horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/SchemaServiceImpl.java#L234)